mirgecom/eos.py

```diff
@@ -310,6 +310,17 @@ def get_internal_energy(self, temperature, massfractions):
         return self._prometheus_mech.get_mixture_internal_energy_mass(
             temperature, massfractions)
 
+    def get_species_molecular_weights(self):
+        """Get the species molecular weights."""
+        return self._prometheus_mech.wts
+
+    def get_production_rates(self, cv: ConservedVars):
+        """Get the production rate for each species."""
+        temperature = self.temperature(cv)
+        return self._prometheus_mech.get_net_production_rates(cv.mass,
+                                                              temperature,
+                                                              cv.massfractions)
+
```

Comment:
These appear to be the only functions we need to access to do Prometheus-based chemistry.   They are implemented on the EOS here.  We could have a separate chemistry object - but it seems natural to encapsulate all of the Prometheus mechanism in a single object.